keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -152,11 +150,7 @@ class ApplicationService(
         null
       },
       locations = if (spec is Locatable<*>) {
-        SimpleLocations(
-          account = (spec as Locatable<*>).locations.account,
-          vpc = (spec as Locatable<*>).locations.vpc,
-          regions = (spec as Locatable<*>).locations.regions.map { SimpleRegionSpec(it.name) }.toSet()
-        )
+        (spec as Locatable<*>).locations
```

Comment:
It turns out that were converting between `Locations` types here unnecessarily, since the type in `ResourceSummary` is `Locations<*>`. For the new plugin, I'm planning to make it `Locatable<SimpleRegions>`, and so the resource summary would just have `locations.regions` in the output.

Cc: @erikmunson